doc/source/train/user-guides/experiment-tracking.rst

```diff
@@ -233,11 +242,12 @@ Refer to the tracking libraries' documentation for semantics.
 .. To conveniently link back to Ray Train run, you may want to log the persistent storage path 
 .. of the run as a config.
 
-.. .. code-block:: python
+.. 
```

Comment:
```suggestion
```

Reply:
This code was commented-out before.

@xwjiang2010 I'm wondering if we should leave the commented-out section or just remove it. What do you think? 

https://github.com/ray-project/ray/blob/837ec26326a25ee9b8ea35aed5aa3f511f1cc972/doc/source/train/user-guides/experiment-tracking.rst#L233-L241

Reply:
This is per @matthewdeng 's request to show people how to link back the ray result directory (storage path). However, the API was not quite thought out yet, so Matt asked me to show how to do it when it's ready and hide it for now.

Reply:
Ah, okay. I think we can just leave it then